### PR TITLE
(chore) remove unused ga code

### PR DESCRIPTION
**Pull request by dushan-madetech**
## Jira ticket URL:
https://dfedigital.atlassian.net/browse/TEVA-165?atlOrigin=eyJpIjoiYTM3ZTY4MGJhZjk5NGNmYzg1M2VkNzliMDhmMWQzYzciLCJwIjoiaiJ9
https://dfedigital.atlassian.net/browse/TEVA-164?atlOrigin=eyJpIjoiZDlhZjUxMWYzYmJmNGMxOWJmMTZmODQ1M2Y2Zjc4MTYiLCJwIjoiaiJ9

## Changes in this PR:

- Remove unused Google Analytics code
- Remove unused Terraform variables

## Next steps:

- [ ] Terraform deployment required

### CODE REVIEW

File: app/views/shared/_gtag.html.haml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-%script{async: true,  src: "https://www.googletagmanager.com/gtag/js?id=#{ENV['GOOGLE_ANALYTICS']}"}
-
-:javascript
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-  gtag('config', "#{ENV['GOOGLE_ANALYTICS']}", { 'optimize_id': "#{ENV['GOOGLE_OPTIMIZE_ID']}", 'anonymize_ip': true, 'linker': {
```
Comment (adrianclay) [inline app/views/shared/_gtag.html.haml]: Will Google Optimize continue to function?
Will the "accept_incoming" continue to function?
Will ips continue to be anonymized?
If we're removing these environment variables from there are they used elsewhere?  Can we remove them entirely from the app including terraform etc?

Comment (dushan-madetech) [reply app/views/shared/_gtag.html.haml]: Google optimize will continue to function
🤷‍♂ 
Yes, this was configured in Tag Manager
They're not used elsewhere so you're right that we should remove them from Terraform

Comment (dushan-madetech) [reply app/views/shared/_gtag.html.haml]: The accept_incoming has been set up on google tag manager

Comment (dushan-madetech) [reply app/views/shared/_gtag.html.haml]: Terraform vars removed in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1032/commits/47198cf32e83bfb5b37c56280067f046aae5450a

**Comment by Renny4Real** [comment]
Would add next steps to the PR description, e.g do I need to run terraform plan on staging, production?

**Pull request merged**